action/protocol/staking/handlers.go

```diff
@@ -62,7 +64,15 @@ func (p *Protocol) handleCreateStake(ctx context.Context, act *action.CreateStak
 		return nil, errors.Wrapf(err, "failed to store account %s", actionCtx.Caller.String())
 	}
 
-	receipt, err := p.settleAction(ctx, sm, gasFee)
+	log := &action.Log{
+		Address:     p.addr.String(),
+		Topics:      nil,
```

Comment:
i am thinking we should also define diff topics for each type of staking/register action?
then from the log we can know what action it is
and data field may take diff meaning for diff action

Reply:
We can add when we need to. So far I think we can leave it blank.

Reply:
and add logs for other actions?

like we discussed, a valid candidate try to update with another's existing name, this should end up with a receipt of log with status = failure, but the block can still be created